apps/src/sites/studio/pages/jotformEmbed.js

```diff
@@ -0,0 +1,95 @@
+/**
+ * @file Entry point for JotForm embedded iframe.
+ */
+var formId = document
+  .getElementById('jotform-embed')
+  .getAttribute('data-formId');
+var ifr = document.getElementById(`JotFormIFrame-${formId}`);
+
+if (window.location.href && window.location.href.indexOf('?') > -1) {
+  var get = window.location.href.substr(window.location.href.indexOf('?') + 1);
+  if (ifr && get.length > 0) {
+    var src = ifr.src;
+    src = src.indexOf('?') > -1 ? src + '&' + get : src + '?' + get;
+    ifr.src = src;
+  }
+}
+window.handleIFrameMessage = function(e) {
+  if (typeof e.data === 'object') {
+    return;
+  }
+  var args = e.data.split(':');
+  var iframe;
+  if (args.length > 2) {
+    iframe = document.getElementById('JotFormIFrame-' + args[args.length - 1]);
+  } else {
+    iframe = document.getElementById('JotFormIFrame');
+  }
+  if (!iframe) {
+    return;
+  }
+  switch (args[0]) {
+    case 'scrollIntoView':
+      iframe.scrollIntoView();
+      break;
+    case 'setHeight':
+      iframe.style.height = args[1] + 'px';
+      // The setHeight message seems to be sent every time we load the form, so use this
+      // to fire the hook we are monitoring to indicate the iframe content has loaded.
+      // eslint-disable-next-line
+      JotFormFrameLoaded();
```

Comment:
is it ok for the monitoring if this is called more than once per load?

Reply:
I think that would give us misleading stats about how often this happens.  I'd fix this in jotformLoader.js though, where we listen for this call, and be resilient against multiple calls from either embed method in the future.

Reply:
Actually, I suppose that promise can't resolve twice... I wonder if it causes an error.

Reply:
It seems to resolve twice (or at least it logs the call twice)
<img width="422" alt="Screen Shot 2019-04-18 at 6 22 16 PM" src="https://user-images.githubusercontent.com/224089/56400130-ffa77080-6206-11e9-8886-0fa7dd4f1f42.png">

Reply:
update - it only goes into the `then` once, so it only actually sends the load event to New Relic once, but it calls `JotFormFrameLoaded` (which does the `console.log`) twice.

Reply:
Cool, that answers the question.  :+1: